src/safeposix/cage.rs

```diff
@@ -84,7 +84,11 @@ pub struct Cage {
     pub cageid: u64,
     pub cwd: interface::RustLock<interface::RustRfc<interface::RustPathBuf>>,
     pub parent: u64,
-    pub filedescriptortable: interface::RustLock<FdTable>
+    pub filedescriptortable: interface::RustLock<FdTable>,
+    pub getgid: interface::AtomicI32,
+    pub getuid: interface::AtomicI32,
+    pub getegid: interface::AtomicI32,
+    pub geteuid: interface::AtomicI32
```

Comment:
You should replace all these AtomicI32 with RustAtomicI32 when you change the import

Reply:
Done!